pyqtgraph/graphicsItems/CurvePoint.py

```diff
@@ -91,6 +91,9 @@ def paint(self, *args):
         pass
     
     def makeAnimation(self, prop='position', start=0.0, end=1.0, duration=10000, loop=1):
+        # automatic encoding when QByteString expected was removed in PyQt v5.5
+        if not isinstance(prop, bytes):
+            prop = prop.encode('latin-1')
```

Comment:
Can you add a comment here referencing qbytearray.sip and briefly explaining the pyqt API change from 5.4 to 5.5?